### DISCUSSION TITLE
Restore Nested Reblogs/Blockquotes

### DISCUSSION THREAD

**Discussion by gayriku** [AprilSylph/XKit-Rewritten#1433] [Suggestion Box]
idk maybe its just me, but i kinda miss the old reblog/blockquote style (as seen [here](https://new-xkit-extension.tumblr.com/post/131189575609/it-is-finally-time-the-extension-a-lot-of-you))

would it be possible to add an option to reenable it?

**Comment by AprilSylph** [comment]
https://github.com/AprilSylph/XKit-Rewritten/wiki/Excluded-features#reblog-display-options

**Comment by marcustyphoon** [comment]
I might try making a user script that tries at some point for fun but I wouldn't maintain it and it probably wouldn't handle every type of post content so it wouldn't go in this project (in an ideal world, everything in XKit Rewritten... works, I would think). A real version of that feature existing would probably necessitate an enthusiastic developer who uses it personally all of the time and will be motivated to spend time on it regularly.
